paasta_tools/deployd/master.py

```diff
@@ -29,6 +29,33 @@
 from paasta_tools.utils import ZookeeperPool
 
 
+class BounceQueue(PaastaQueue):
```

Comment:
👍 to the better naming.

The inbox queue actually can have multiple copies of the same service instance with different times that they should bounce by. Then the inbox thread de-dupes when it reads off that queue and only stores service instances once and only overwrites when they are being bumped in priority so to speak.